app/controllers/v0/letters_controller.rb

```diff
@@ -8,11 +10,16 @@ def index
              meta: response.metadata
     end
 
-    # :nocov:
     def show
-      head :ok
+      unless EVSS::Letters::Letter::LETTER_TYPES.include? params[:id]
+        raise Common::Exceptions::RecordNotFound, params[:id]
```

Comment:
This gets raised in the service if the particular letter isn't found by type
```ruby
raise Common::Exceptions::RecordNotFound, params[:id] if response.status.to_i == 404
```
However, in this spot, we're dealing with an invalid parameter which may warrant a different exception.  Otherwise, there could be ambiguity if we end up having to look at logs.